app/assets/stylesheets/comments.scss

```diff
@@ -0,0 +1,21 @@
+.new-comment-block{
+  border:1px solid black;
+  border-top: 0px;
+  border-bottom: 0px;
+  border-radius: 10px;
+  padding:1%;
+  margin-bottom: 2%;
+  input{
+    margin:1%;
+  }
+}
+
+.comment-block{
+  margin-bottom:2%;
+}
+
+.comments-header{
+  margin-top:2%;
+  text-size:1rem;
+  border-bottom: 1px solid black;
```

Comment:
Šeit pat redzams piemērs ar nevienādu koda formatēšanu. Pirmais solis ir - visur izmantot pēc iespējas vienātu koda formatēšanu! Ievēro 19 rindiņ;ā nav space `starp text-size:1rem;`, bet 20 rindiņā jau ir atstarpe.

Tu vari pat savā personīgajā projektā pieturēties pie kāda stila gribi, bet svarīgi tad to arī ievērto!